phpunit/block-supports/typography-test.php

```diff
@@ -283,6 +283,111 @@ public function test_should_generate_classname_for_font_family() {
 		$this->assertSame( $expected, $actual );
 	}
 
+	/**
+	 * Tests that stabilized typography supports will also apply to blocks using
+	 * the experimental syntax, for backwards compatibility with existing blocks.
+	 *
+	 * @covers ::wp_apply_typography_support
+	 */
+	public function test_should_apply_to_experimental_typography_supports() {
```

Comment:
```suggestion
	public function test_should_apply_experimental_typography_supports() {
```

I might be misreading the intent of the name but the tests seems related to the application of supports rather than applying something to the supports themselves. Ignore me if I'm getting it wrong.

Reply:
Not misreading anything! I tried out a couple of different names for these tests and this was a typo when settling on the final one 😄